server/app/models/grid_service_health_check.rb

```diff
@@ -10,4 +10,11 @@ class GridServiceHealthCheck
 
   embedded_in :grid_service
 
-end
\ No newline at end of file
+  def is_valid?()
```

Comment:
`valid?` clashes with Mongoid hence is used ìs_valid?` instead. But I'll remove the unnecessary brackets.